packages/web3-eth-contract/test/integration/setup.js

```diff
@@ -19,6 +19,6 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 require('../config/setup');
 
-const jestTimeout = 15000;
+const jestTimeout = 20000;
```

Comment:
Instead of `jestTimeout` def in multiple test files, shouldn't we have one used by all provided by cli --testTimeout .

Reply:
I've reverted all timeouts to 15 sec except web3-eth